ui/main.go

```diff
@@ -7,15 +7,15 @@ import (
 	"github.com/fatih/color"
 )
 
-var (
-	// Silent prevents any output from printing.
-	Silent = false
-
-	// Quiet removes all additional formatting
-	Quiet = false
+const (
+	logLevelSilent  = iota
```

Comment:
Love the idea to use an `iota` here for levels.

This could be combined with a custom type for safety as well:

```go
type Level uint8

const (
	logLevelSilent Level = iota
	// ...
)
```